packages/bits/demo/src/components/demo/wizard-v2/wizard-remove-step/wizard-remove-step-example.component.ts

```diff
@@ -5,14 +5,14 @@ import {
     ViewChild,
 } from "@angular/core";
 
-import {IWizardStepConfig } from "../wizard-dynamic/wizard-dynamic.example.component";
+import {IWizardStepV2Component } from "../wizard-dynamic/wizard-dynamic.example.component";
```

Comment:
I would be good to avoid using a mystery interface from another example. Can we duplicate the interface in this example, so that the code is self-contained?

Reply:
That's a good catch, maybe a leftover from previous commits. Fixed